src/components/common/TmSessionExplore.vue

```diff
@@ -135,12 +135,13 @@ export default {
         return false
       }
     },
-    isNotAnyOtherCosmosAddress(param) {
+    isAWhitelistedBech32Prefix(param) {
       if (
-        param.substring(0, 9) !== "cosmospub" &&
-        param.substring(0, 13) !== "cosmosvalcons" &&
-        param.substring(0, 16) !== "cosmosvalconspub" &&
-        param.substring(0, 16) !== "cosmosvaloperpub"
+        param.substring(0, 7) === "cosmos1" ||
```

Comment:
@Bitcoinera @faboweb this is another place where we should think about how to reduce frontend changes for new networks...

Reply:
#underengineered @faboweb 😝